backend/src/backend/exporters/kiwix_public_exporter.py

```diff
@@ -0,0 +1,38 @@
+import pathlib
+import tempfile
+
+import openzim_uploader
+
+from backend.collections.kiwix_public import KiwixPublicCollection
+from backend.constants import BackendConf, logger
+from backend.exporters import ExporterInterface
+from backend.formatters.kiwixlibraryxml import KiwixLibraryXml
+
+
+class KiwixPublicExporter(ExporterInterface):
+    """Upload XML Kiwix Library to"""
```

Comment:
to? 
```py
    """ Upload XML Kiwix Library to `UPLOAD_URI`, using `PRIVATE_KEY`"""
```

Given #92, it might be good to use a more-specific environ and variable name